### ISSUE TITLE
Send Evaluators an email notification when they are invited to join a challenge phase

### ISSUE THREAD

**Issue by stepchud** [GSA/Challenge_platform#231]
# What/Why
Blocked by #41 
Evaluators should receive an email notification when they are invited to a challenge phase.
The template is not designed yet.

### Figma Link

[link-to-figma](https://www.figma.com/design/VfdBFG8ShVnOStdSNYy9fR/Challenge.Gov-Component-Library)

## Acceptance Criteria
- [ ] A requirement for completion
  - [ ] Sub-requirement (optional)
  - Additional notes (optional)

_Disregard the following if this work **does not** include any UI changes_
- [ ] WAVE screenshot
- [ ] Automated a11y tests (axe-rspec specs)
- [ ] Keyboard accessibility (tab navigation, form interactions)
- [ ] Responsiveness at desktop and mobile

**Comment by stepchud** [comment]
Duplicate of #325
